integration-tests/ahm/src/tests.rs

```diff
@@ -470,13 +484,38 @@ async fn migration_works_time() {
 	run_check(|| RcChecks::post_check(rc_pre.clone().unwrap()), &mut rc);
 
 	// Post-checks on the Asset Hub
-	run_check(|| AhChecks::post_check(rc_pre.unwrap(), ah_pre.unwrap()), &mut ah);
+	run_check(|| AhChecks::post_check(rc_pre.clone().unwrap(), ah_pre.clone().unwrap()), &mut ah);
 
 	println!(
 		"Migration done in {} RC blocks, {} AH blocks",
 		rc_block_end - rc_block_start,
 		ah_block_end - ah_block_start
 	);
+
+	// run the migration again to check its idempotent
+
+	// Set the initial migration stage from env var if set.
+	set_initial_migration_stage(
+		&mut rc,
+		Some(RcMigrationStage::AccountsMigrationOngoing { last_key: None }),
```

Comment:
we do not wanna run the accounts init that includes the balances record initialization since it is not idempotent